src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -174,26 +174,29 @@
 
 @snippet optionalArrayMethodSampleCodeBidiStreaming(apiMethod, initCode)
   requests = [request]
-  {@responseStreamingSampleCode(apiMethod, initCode)}
+  {@responseStreamingSampleCode(apiMethod, initCode, processElement())}
 @end
 
-@snippet pagedOptionalArrayMethodSampleCodeAll(apiMethod, initCode)
+@snippet pagedOptionalArrayMethodSampleCodeAll(apiMethod, initCode, processResp)
   @# Iterate over all results
-  {@responseStreamingSampleCode(apiMethod, initCode)}
+  {@responseStreamingSampleCode(apiMethod, initCode, processResp)}
 @end
 
-@snippet pagedOptionalArrayMethodSampleCodePaged(apiMethod, initCode)
+@snippet pagedOptionalArrayMethodSampleCodePaged(apiMethod, initCode, processResp)
   @# Iterate over results one page at a time
   for page in {@pagedMethodCallSampleCode(apiMethod, initCode)}:
       for element in page:
-          @# process element
-          pass
+          {@processResp}
 @end
 
-@private responseStreamingSampleCode(apiMethod, initCode)
+@private responseStreamingSampleCode(apiMethod, initCode, processResp)
   for element in {@methodCallSampleCode(apiMethod, initCode)}:
-      @# process element
-      pass
+      {@processResp}
+@end
+
+@private processElement()
```

Comment:
This may need a more descriptive name once we process the LRO responses, but we can deal with that then.